Modules/Core/Common/include/itkDefaultVectorPixelAccessorFunctor.h

```diff
@@ -77,7 +77,7 @@ class DefaultVectorPixelAccessorFunctor
   }
 
 
-  DefaultVectorPixelAccessorFunctor () : m_Begin(nullptr) {}
+  DefaultVectorPixelAccessorFunctor() = default;
```

Comment:
This is missing `ITK_NOEXCEPT`. Is that on purpose?

Reply:
@dzenanz I would have liked to add `ITK_NOEXCEPT` to this specific default-constructor as well, but I hesitated, because it was not 100% clear to me if its data member `DefaultVectorPixelAccessorFunctor::m_PixelAccessor` could possibly have a throwing default-constructor. So I thought: when in doubt, don't noexcept!